cluster/gce/configure-vm.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -x
```

Comment:
I am adding this on purpose. configure-vm.sh runs on every gce instances at boot time. Once it went wrong, it is very difficult to debug. This only write to startupscript.log. I hope it is ok.